### PR TITLE
Wip development

### PR DESCRIPTION
**Pull request by markedwardmurray**
@jmburges so this became pretty long too. I can move the content about `BOOL` and `CGFloat` until later in the curriculum if you want me to streamline it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,343 @@
...
+  * `NSInteger i = 0;`
+
+2. Use an `NSLog()` with the `%li` format specifier to print `i` to the console:
+  * `NSLog(@"i: %li", i);`
+  * Copy your `NSLog()` to your clipboard with `⌘` `C`.
+  * Run the program with `⌘` `R` to see that `i: 0` prints to the console.
+
+3. Redefine `i` to the result of adding `1` and `1` and print `i` to the console again:
+  * `i = 1 + 1;`
+  * Use `⌘` `C` to paste your `NSLog()` a second time after the redefinition of `i`.
```
Comment (misterfifths) [inline README.md]: C -> V,
and "redefinition" is misleading. "modification"?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,343 @@
...
+  * Run the program with `⌘` `R` to see that `i: 2` prints to the console.
+
+4. Redefine `i` again to the result of various simple calculations. Paste in a new copy of the `NSLog()` after each redefinition of `i` so you can see each change in its value in the debug console.
```
Comment (misterfifths) [inline README.md]: same feelings re: redefinition

File: README.md (hunk 1 above)
Comment (markedwardmurray) [inline README.md]: haha, oops.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,343 @@
...
+
+4. Now compare the result of calculating the square root of `2` with `math.h`'s definition of it `M_SQRT2`:
+  * `f = M_SQRT2;`
+  * How does the definition compare to the calculation?
+
+5. Use the `sqrt(x)` function to find the square root of 81.
+
+6. Now use the `pow(x, y)` function to find the cube of 3.
+
+7. Finally, use the `exp2(x)` function to find the highest value of a 64-bit integer.
```
Comment (misterfifths) [inline README.md]: Huh? How are they supposed to use exp2 for that?

Comment (markedwardmurray) [inline README.md]: `exp2(64)` returns 2^64 ?

Comment (misterfifths) [inline README.md]: Hm ok.. exp2 returns a float, which is why I was confused. I guess that's ok.. though the maximum value of a signed 64-bit integer is actually 2^32-1...

Comment (markedwardmurray) [inline README.md]: yeah i got really tired of this code-along by the end of it, i wasn't thinking things through.

**Comment by markedwardmurray** [comment]
regarding https://github.com/learn-co-curriculum/mobile-development-with-ios-track/issues/9

**Pull request merged**